src/screens/CreatePost/index.js

```diff
@@ -778,6 +756,8 @@
                 setIsPollMultipleChoice(ismultiplechoice)
               }
               expiredobject={postExpired[expiredSelect].expiredobject}
+              isAnonym={typeUser}
+              expiration={postExpired[expiredSelect].label}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isAnonym` prop is now being passed to the `CreatePollContainer` component. Make sure this prop is being correctly used within the component for conditional rendering or styling changes.
```diff
+ isAnonym={typeUser}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->